server/constants/vars.js

```diff
@@ -8,6 +8,8 @@ export const INCLUSION_EXCLUSION_CRITERIA_FORM =
 export const SOCIODEMOGRAPHICS_FORM = 'form_sociodemographics'
 export const TOTAL_LABEL = 'Total'
 export const SITE = 'Site'
+export const site = 'site'
```

Comment:
we shouldn't have a lowercase constant here. Why do we need 2 constants for the same word (site and sites)?

Reply:
[5b4b2a4](https://github.com/AMP-SCZ/dpdash/pull/695/commits/5b4b2a4823862cecfb04b595c18aff35b4b2c9b0)